pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -16,11 +16,11 @@ with stdenv.lib;
 buildPythonApplication rec {
 
   name = "youtube-dl-${version}";
```

Comment:
`pname`

Reply:
@dotlambda What do you mean?

Reply:
You can drop the `name` and set `pname = "youtube-dl". This makes it easier to override the version.

Reply:
I tried that, but it doesn't build:
```
error: undefined variable 'name' at /home/bart/source/nixpkgs/pkgs/tools/misc/youtube-dl/default.nix:22:53
```

Reply:
That's because `${name}` is used in the url. You'll have to replace it by `${pname}-${version}`.

Reply:
Thanks, done.